src/init/features/dataconnect/index.ts

```diff
@@ -74,22 +74,29 @@ const defaultSchema = { path: "schema.gql", content: SCHEMA_TEMPLATE };
 
 // doSetup is split into 2 phases - ask questions and then actuate files and API calls based on those answers.
 export async function doSetup(setup: Setup, config: Config): Promise<void> {
-  const info = await askQuestions(setup);
+  const isBillingEnabled = setup.projectId ? await checkBillingEnabled(setup.projectId) : false;
```

Comment:
Optional, canbe done later: Checking billing status is probably appropriate to do in init/index.ts and then passed around in setup, since it won't change from product to product.

Reply:
Already passed this into `askQuestions`.

Only called once